lib/runtime/invalid_transaction_test.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2022 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package runtime
+
+import (
+	"testing"
+
+	"github.com/ChainSafe/gossamer/lib/transaction"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestInvalidTransactionErrors(t *testing.T) {
```

Comment:
nit NEVER use snake case, but for Test function names such that it's `Test_Struct_Method` or `Test_Function`.

Camel case is rather confusing to read in code and in logs for test names, and using snake case like so really helps imo. 😉 

```suggestion
func Test_InvalidTransaction_Error(t *testing.T) {
```